query.js

```diff
@@ -0,0 +1,6 @@
+function BrowserslistQuery (type, value) {
```

Comment:
Do we need a separated file just for the simple class without methods? We can use plain object and reduce project size

Reply:
Well, I added it because there is 2 places here I currently need the `BrowserslistQuery` so I don't break old code. `function resolve (queries, context) {` can also take a array of strings.

+ https://github.com/browserslist/browserslist/pull/286/files#diff-168726dbe96b3ce427e7fedce31bb0bcR300
+ https://github.com/browserslist/browserslist/pull/286/files#diff-168726dbe96b3ce427e7fedce31bb0bcR139

If I can avoid code duplication (multiple places where I initialize the same object) in my re-write, then I'll remove `BrowserslistQuery`.